src/Catalyst.Core.Lib/CoreLibProvider.cs

```diff
@@ -164,5 +166,11 @@ protected override void Load(ContainerBuilder builder)
 
             base.Load(builder);
         }
+
+        // TODO: rethink validation of transaction signature
+        class AlwaysTrueTransactionValidator : ITransactionValidator
+        {
+            public bool ValidateTransaction(PublicEntry transaction) => true;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the unused private method 'ValidateTransaction'.](https://app.codacy.com/manual/catalyst-network/Catalyst.Framework/pullRequest?prid=4979613)